jrnl/jrnl.py

```diff
@@ -49,6 +50,11 @@ def run(args):
             args=args, config=config, original_config=original_config
         )
 
+    # Apply config overrides
+    overrides = args.config_override
+    if overrides:
+        config = apply_overrides(overrides, config)
```

Comment:
This should be farther up in the file (right after line 40 `config = scope_config(config, args.journal_name)`) since there are commands that require the config. If the overrides stay here, then none of those commands (import, encrypt, decrypt, etc) will work with overrides.

Reply:
good to know, and thanks for catching that. Sounds like behavior I should have tested, so I can add some